### PR TITLE
Bluebird changes

### PR DESCRIPTION
**Pull request by renujhamtani**
@vrathee @gandhikeyur Please review these changes

### CODE REVIEW

File: uds.js [hunk 1]
```diff
@@ -181,119 +136,71 @@
...
     };
-
-    uds.postPrivateComments = function (onSuccess, onFailure, caseNumber,caseComment,hoursWorked) {
-        if (!$.isFunction(onSuccess)) {
-            throw 'onSuccess callback must be a function';
-        }
-        if (!$.isFunction(onFailure)) {
-            throw 'onFailure callback must be a function';
-        }
-        if(hoursWorked===undefined){
```
Comment (vrathee) [inline uds.js:212]: @renujhamtani Why did you remove the hoursWorked logic?

Comment (renujhamtani) [inline uds.js:212]: @vrathee I think my bad...merging conflict i am doing it right now

**Comment by renujhamtani** [comment]
@sshumake @vrathee Please review. I have merged all changes. Please do not merge this until the prod push.

File: uds.js [hunk 2]
```diff
@@ -181,157 +136,116 @@
...
+        return executeUdsAjaxCall(url,'GET');
     };
 
-    uds.removeUserSbr = function (onSuccess, onFailure,userId,query) {
-        if (!$.isFunction(onSuccess)) { throw 'onSuccess callback must be a function'; }
-        if (!$.isFunction(onFailure)) { throw 'onFailure callback must be a function'; }
-        var url = udsHostName.clone().setPath('/user/'+userId+'/sbr').addQueryParam('where', query);
-        return executeUdsAjaxCall(onSuccess,onFailure,url,'DELETE');
+    uds.getSbrDetails = function (sbrName) {
+        var url = udsHostName.clone().setPath('/user?resourceProjection=Full&where=(sbrName is "' + sbrName + '" or roleSbrName is"' + sbrName + '" )');
```
Comment (vrathee) [inline uds.js:213]: This function doesn't exist in latest uds.js.. need to take care during merge.

**Comment by vrathee** [comment]
Apart from 1 comment rest looks great..

**Comment by sshumake** [comment]
@renujhamtani looks good! @engineersamuel FYI udsjs has migrated to bluebird

**Comment by engineersamuel** [comment]
awesome, thank you @renujhamtani !
